keps/prod-readiness/sig-auth/3225.yaml

```
@@ -0,0 +1,3 @@
+kep-number: 3225
+alpha:
+  approver: "@enj"
```

```suggestion
  approver: "@deads2k"
```

Has to be one of:

https://github.com/kubernetes/enhancements/blob/9b475dc714197980b2d12dea2800f256be1f0e74/OWNERS_ALIASES#L187-L190